src/Utils/useFileUpload.tsx

```diff
@@ -414,11 +420,9 @@ export default function useFileUpload(
         onChange={onFileChange}
         type="file"
         accept={
-          "allowAllExtensions" in options
-            ? "image/*,video/*,audio/*,text/plain,text/csv,application/rtf,application/msword,application/vnd.oasis.opendocument.text,application/pdf,application/vnd.ms-excel,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet,application/vnd.oasis.opendocument.spreadsheet,application/pdf"
-            : "allowedExtensions" in options
-              ? options.allowedExtensions?.join(",")
-              : ""
+          "allowExtensions" in options
```

Comment:
It was a good practice to require `allowAllExtensions` to be explicit when using the Component, rather than using all extensions as default